src/main/java/com/alkemy/ong/mapper/UserMapper.java

```diff
@@ -59,4 +60,10 @@ public static User mapUpdateDTOToDomain(UserUpdateDTO updateDTO){
                 .photo(updateDTO.getPhoto()).build();
         return userDomain;
     }
+
+    public static User mapLoginDTOToDomain(UserLoginDTO userLoginDTO) {
+        User user = User.builder().password
```

Comment:
Corregir formato acá, despues del builder() deberíamos tener un campo por renglón:
```suggestion
        User user = User.builder()
        .password(...)
        .email(...)
        .build();
```